istio-telemetry/kiali-operator/values.yaml

```diff
@@ -0,0 +1,71 @@
+#
+# addon kiali
+#
+kiali:
+  replicaCount: 1
+  hub: docker.io/kiali
```

Comment:
At some point in the near future, Kiali team is going to stop publishing on docker.io. All Kiali images are also going up on quay.io which will be the location going forward. Please change the hub to point to quay.io (as opposed to docker.io).

Reply:
Done. Thanks.